src/blueprint/azext_blueprint/tests/latest/test_blueprint_scenario.py

```diff
@@ -105,6 +116,29 @@ def test_blueprint(self, resource_group):
             '--name "{assignmentName}"',
             checks=[])
 
+        with mock.patch('azure.cli.command_modules.role.custom._gen_guid', side_effect=self.create_guid):
+            self.cmd(
+                'az role assignment create '
+                '--assignee-object-id {identityPrincipalId} '
+                '--role Owner '
+                '--scope /subscriptions/{subscription}/resourceGroups/{rgName}')
+
+        import time
+        time.sleep(600)

```

Comment:
Better to do it this way:
```suggestion
        if self.is_live:
            time.sleep(600)

```

Reply:
According to @qwordy, `time.sleep()` is automatically mocked in playback test.
https://github.com/Azure/azure-cli/blob/00cf5e3b3ecadfa43d4b1bee8c5c2d352626d227/src/azure-cli-testsdk/azure/cli/testsdk/base.py#L108

Reply:
@fengzhou-msft Got it! It's great!